check/Util.hs

```diff
@@ -1,6 +1,6 @@
 module Util where
 
-import           System.Process
+import System.Process
```

Comment:
I think aligning import statements like this is pretty common by now (and yes, contrary to what I said above, it's not a multiple of two).  I don't really care, I just adopted it at some point as others did it.